### PR TITLE
functionality to delete comments added

### PR DESCRIPTION
**Pull request by nickkagame**

### CODE REVIEW

File: controllers.js [hunk 1]
```diff
@@ -80,5 +81,13 @@ exports.sendComment = (request, response, next) => {
...
     response.status(200).send({ users });
-  });
+  })
+  .catch(next)
 }
+
+exports.deleteComment = (request, response, next) => {
+  const {comment_id} = request.params
+  eraseComment(comment_id).then((deletedComment) => {
+    response.status(204).send(deletedComment)
```
Comment (alexdevdavis) [inline controllers.js:91]: perhaps surprisingly, the ticket asks that you return no content at all, so `.send()` 👍

File: models.js [hunk 2]
```diff
@@ -141,3 +141,13 @@ exports.fetchUsers = () => {
     })
   }
 
+
+  exports.eraseComment = (comment_id) => {
+    const query = `DELETE FROM comments WHERE comment_id = $1
+    RETURNING *;`
+    return db.query(query, [comment_id])
+    .then((comments) => {
+      console.log(comments.rows[0])
```
Comment (alexdevdavis) [inline models.js:150]: can delete this console log, but you do want some way of checking that the delete was successful; a condition in this .then block will do

File: __tests__/app.test.js [hunk 3]
```diff
@@ -497,5 +497,12 @@ describe("APP", () => {
       });
     });
   });
+  describe('12 DELETE COMMENTS', () => {
+    test('will return a 204 once completed', () => {
+      return request(app)
+      .delete('/api/comments/3')
+      .expect(204)
+    })
+  });
```
Comment (alexdevdavis) [inline __tests__/app.test.js:506]: error handling for delete endpoint?

**Review by alexdevdavis — APPROVED** [review]
DELETE is a nice one, isn't it 👍 
- a couple of pointers regarding success and fail states
- have a think about error handling for this one

File: __tests__/app.test.js (hunk 3 above)
Comment (nickkagame) [reply __tests__/app.test.js:506]: sorted this now!

**Pull request merged**